roles/mirror/tasks/route53.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Create AWS config dir
+  ansible.builtin.file:
+    path: /root/.aws
+    state: directory
+    mode: "0700"
```

Comment:
Surprised ansible-lint didn't complaint about not explicitly settings the owner/group as well?

Reply:
It can't really be wrong here.  Only root can write to /root.  Maybe that's why?